core/qt-ble.cpp

```diff
@@ -154,6 +154,7 @@ static const char *match_uuid_list(const QBluetoothUuid &match, const struct uui
 // Oh. It did, didn't it?
 //
 static const struct uuid_match serial_service_uuids[] = {
+	{ "6e400001-b5a3-f393-e0a9-e50e24dc10b8", "Cressi"},
```

Comment:
I think we need a comment here to explain the situation, to allow future developers to understand that this has to be higher priority than Nordic Semi UART.

Reply:
Done. I also moved the Nordic UART to the end. PTAL